src/Services/Processor/Substrate/Codec/Encoder.php

```diff
@@ -262,7 +263,12 @@ public function getEncoded(string $type, array $params): string
             ...$params,
         ]);
 
-        return HexConverter::prefix($encoded);
+        $hex = HexConverter::prefix($encoded);
+        if (preg_match('/^0x[a-fA-F0-9]+$/', $hex) >= 1) {
+            return $hex;
+        }
```

Comment:
I believe we have this helper
```suggestion
        if (Hex::isHexEncoded($hex) {
            return $hex;
        }
```